core/src/main/java/org/elasticsearch/action/admin/cluster/reroute/ClusterRerouteRequestBuilder.java

```diff
@@ -59,12 +58,4 @@ public ClusterRerouteRequestBuilder setExplain(boolean explain) {
         request.explain(explain);
         return this;
     }
-
-    /**
-     * Sets the commands for the request to execute.
-     */
-    public ClusterRerouteRequestBuilder setCommands(AllocationCommand... commands) throws Exception {
-        request.commands(commands);
-        return this;
-    }
```

Comment:
don't we have to replace this setter with the new corresponding one that we have in the request?